idd/Energy+.idd.in

```diff
@@ -18672,120 +18677,217 @@ SurfaceProperty:SurroundingSurfaces,
        \object-list ScheduleNames
        \note Schedule values are real numbers, -100.0 to 100.0, units C
        \note optional
-   N2, \field Ground View Factor
+   A3, \field Surrounding Surface 1 Name
+       \begin-extensible
+       \required-field
+       \type alpha
+   N2, \field Surrounding Surface 1 View Factor
+       \required-field
        \minimum 0.0
        \maximum 1.0
-       \default 0.5
-       \note optional
-   A3, \field Ground Temperature Schedule Name
```

Comment:
The fields `Surrounding Surface 1 View Factor` is initialized to -1.0 if left blank. No need to have a default value.